plutus-benchmark/marlowe/README.md

```diff
@@ -1,5 +1,7 @@
 # Experimental version of Marlowe validator for Cardano, with minimal dependencies
 
+## `marlowe-internal`
+
 This package is fully representative version of the Marlowe validator on chain. It is primarily for benchmarking/profiling Marlowe scripts. 
```

Comment:
> This package is fully representative version of the Marlowe validator on chain.

Is it going to stay this way?

Reply:
Brian said that the on chain validator won't change for a while but good point, I'll make it more clear.